src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/TdsParserHelperClasses.cs

```diff
@@ -156,7 +156,7 @@ internal sealed class SqlFedAuthToken
 
         internal SqlFedAuthToken(
             byte[] accessToken,
-            uint expirationFileTime)
+            long expirationFileTime)
```

Comment:
Will this also affect the current codebase - seem to have seen a few issues relating to to many token renewals?